lib/cc/engine/analyzable_files.rb

```diff
@@ -27,13 +26,17 @@ def fetch_files(paths)
       end
 
       def build_files_with_inclusions(inclusions)
-        fetch_files(inclusions)
+        if inclusions == ["./"]
```

Comment:
I'm trying to understand why `./` is a special case. What if I `analyze ./foo`, how does that not cause similar problems?